### DISCUSSION TITLE
Sticky Header for DaisyUI

### DISCUSSION THREAD

**Discussion by myhendry** [saadeghi/daisyui#1861] [Q&A]
Hi. Is there any sticky header/navbar for DaisyUI? Thanks

**Comment by fini** [comment]
Not out of the box, but you could easily turn it into one using standard CSS/Tailwind and JS.

**Reply by myhendry** [reply]
Thanks. I found it. I will post the answer below so others can use it next time

**Comment by myhendry** [comment]
to achieve a sticky header/navbar, you can do below...

```
    <header className="navbar sticky top-0 z-30 ..."> ... </header>
```
